src/js/reducers/sessionReducer.jsx

```diff
@@ -90,6 +91,7 @@ export default function (state = initialState, action) {
         isSuperuser: _.get(action, 'payload.data.data.isSuperuser'),
         isUserAdmin: _.get(action, 'payload.data.data.isUserAdmin'),
         isUserApprover: _.get(action, 'payload.data.data.isUserApprover', false),
+        isUserRequestApprover: _.get(action, 'payload.data.data.isUserRequestApprover', false),
```

Comment:
whenever I see this ugly `_.get` I wish we refactored it in the future :pray: 
to something like:
```js
const { isUserRequestApprover, ... } = payload.data.data
...
isUserApprover: isUserApprover ?? false
```